pyproject.toml

```diff
@@ -48,14 +48,16 @@ isort = "^5"
 pre-commit = "2"
 flake8 = "^4"
 mypy = "^0.931"
-cookiecutter = "^1.7.3"
 pytest-cov = "^3.0.0"
 pytest-cookies = "^0.6.1"
 Sphinx = "^4.4.0"
 Flake8-pyproject = "^0.9.0"
 jupyter-book = "^0.12.3"
 myst-parser = "^0.15"
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
@xmnlab seems like poetry now has group dependencies, which I think `--dev` is just one of the possible groups, as you can see a "test" group in this example: https://python-poetry.org/docs/master/managing-dependencies/. But I didn't find any reference to poetry version in the project, so I assume it is correct..

Reply:
Yep I saw that some days ago, but never tried. But for us just dev should be enough